MVCapplication/src/main/resources/ValidationMessages_uk.properties

```diff
@@ -14,4 +14,4 @@ error.productCategory.notempty=\u0412\u0438\u0431\u0435\u0440\u0456\u0442\u044C
 error.productUnit.notempty=\u0412\u0438\u0431\u0435\u0440\u0456\u0442\u044C \u043E\u0434\u0438\u043D\u0438\u0446\u0456 \u0432\u0438\u043C\u0456\u0440\u044E\u0432\u0430\u043D\u043D\u044F \u043F\u0440\u043E\u0434\u0443\u043A\u0442\u0443
 error.categoryName.notNull=\u0412\u0432\u0435\u0434\u0456\u0442\u044C \u0456\u043C'\u044F \u043A\u0430\u0442\u0435\u0433\u043E\u0440\u0456
 error.storage.amount=\u041A\u0456\u043B\u044C\u043A\u0456\u0441\u0442\u044C \u043C\u0443\u0441\u0438\u0442\u044C \u0431\u0443\u0442\u0438 \u0431\u0456\u043B\u044C\u0448\u043E\u044E \u043D\u0456\u0436 {value}
-error.storage.product=\u041A\u0456\u043B\u044C\u043A\u0456\u0441\u0442\u044C \u043C\u0443\u0441\u0438\u0442\u044C \u0431\u0443\u0442\u0438 \u0431\u0456\u043B\u044C\u0448\u043E\u044E \u043D\u0456\u0436 {value}
\ No newline at end of file
+error.storage.product=\u041A\u0456\u043B\u044C\u043A\u0456\u0441\u0442\u044C \u043C\u0443\u0441\u0438\u0442\u044C \u0431\u0443\u0442\u0438 \u0431\u0456\u043B\u044C\u0448\u043E\u044E \u043D\u0456\u0436 {value}
```

Comment:
How do you understand this? Or this is Github issue?

Maybe some issue with file encoding?